2022-03-13 00:01:35,827 INFO    MainThread:14119 [wandb_setup.py:_flush():71] setting env: {}
2022-03-13 00:01:35,828 INFO    MainThread:14119 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-13 00:01:35,828 INFO    MainThread:14119 [wandb_init.py:_log_setup():371] Logging user logs to /home/aiotlab/projects/cuongdm/MEC1/wandb/run-20220313_000135-2rjc03ly/logs/debug.log
2022-03-13 00:01:35,828 INFO    MainThread:14119 [wandb_init.py:_log_setup():372] Logging internal logs to /home/aiotlab/projects/cuongdm/MEC1/wandb/run-20220313_000135-2rjc03ly/logs/debug-internal.log
2022-03-13 00:01:35,828 INFO    MainThread:14119 [wandb_init.py:init():404] calling init triggers
2022-03-13 00:01:35,828 INFO    MainThread:14119 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-03-13 00:01:35,828 INFO    MainThread:14119 [wandb_init.py:init():460] starting backend
2022-03-13 00:01:35,828 INFO    MainThread:14119 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-13 00:01:35,833 INFO    MainThread:14119 [backend.py:ensure_launched():216] starting backend process...
2022-03-13 00:01:35,841 INFO    MainThread:14119 [backend.py:ensure_launched():221] started backend process with pid: 14654
2022-03-13 00:01:35,850 INFO    MainThread:14119 [wandb_init.py:init():469] backend started and connected
2022-03-13 00:01:35,853 INFO    MainThread:14119 [wandb_init.py:init():533] updated telemetry
2022-03-13 00:01:35,890 INFO    MainThread:14119 [wandb_init.py:init():563] communicating current version
2022-03-13 00:01:37,166 INFO    MainThread:14119 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-13 00:01:37,166 INFO    MainThread:14119 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-13 00:01:37,466 INFO    MainThread:14119 [wandb_init.py:init():606] starting run threads in backend
2022-03-13 00:01:42,474 INFO    MainThread:14119 [wandb_run.py:_console_start():1810] atexit reg
2022-03-13 00:01:42,475 INFO    MainThread:14119 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-13 00:01:42,475 INFO    MainThread:14119 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-13 00:01:42,477 INFO    MainThread:14119 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-13 00:01:42,478 INFO    MainThread:14119 [wandb_init.py:init():633] run started, returning control to user process
2022-03-13 00:01:59,559 INFO    MainThread:14119 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-13 00:01:59,563 INFO    MainThread:14119 [wandb_run.py:_restore():1752] restore
2022-03-13 00:02:02,509 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:02:02,775 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:02:03,443 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:02:03,545 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15029
}

2022-03-13 00:02:03,648 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15029
}

2022-03-13 00:02:03,750 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15029
}

2022-03-13 00:02:03,852 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 15029
}

2022-03-13 00:02:03,954 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:04,057 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:04,159 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:04,261 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:04,363 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:04,465 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:05,200 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}

2022-03-13 00:02:06,057 INFO    MainThread:14119 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15029
  total_bytes: 15029
}
local_info {
}

2022-03-13 00:02:07,340 INFO    MainThread:14119 [wandb_run.py:_append_files():2180] logging synced files
